packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -334,6 +334,14 @@ const double _kDefaultFontSize = 14.0;
 /// ```
 /// {@end-tool}
 ///
+/// #### Supported font formats
+///
+/// Font formats currently supported by Flutter :
+/// * `.ttf`
```

Comment:
Flutter also supports TrueType Collections, (`.ttc`) which contain multiple tts files. This is commonly seen in CJK font collections that include multiple fonts for multiple languages.

Reply:
For example, see https://www.google.com/get/noto/help/cjk/ where the truetype downloads are `.ttc` files.

Reply:
@GaryQian I had missed to mention it. Thanks for the tip. I'll make the changes.